ymer/src/main/java/com/avanza/ymer/support/JavaLocalDateTimeReadConverter.java

```diff
@@ -15,12 +15,17 @@
  */
 package com.avanza.ymer.support;
 
-import org.springframework.core.convert.converter.Converter;
-import org.springframework.data.convert.ReadingConverter;
-
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
 
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.data.convert.ReadingConverter;
+
+/**
+ * Converter that converts from a String to a {@link LocalDateTime}. This is a good choice if high time precision
+ * is more important that saving space. Otherwise {@link JavaTimeLocalDateTimeReadConverter} could be
```

Comment:
Typo? "is more important _**than**_ saving space"

Reply:
Good catch! Fixed in a separate commit.